pylint_google_style_guide_imports_enforcing/__init__.py

```diff
@@ -21,27 +21,30 @@ class AlphabeticallySortedImports(BaseChecker):
 
     priority = -1
 
+    excluded_modules = ['typing', 'typing_extensions', 'six.moves']
```

Comment:
Are those specifically excluded in google style guide or you're making your own exceptions? In the latter case, can you make it configurable?

Here you can find examples of how to make config:
1. Define options at class level https://github.com/Enforcer/pylint-forbidden-imports/blob/master/pylint_forbidden_imports/forbidden_imports.py#L44
2. Read it inside `open` method https://github.com/Enforcer/pylint-forbidden-imports/blob/master/pylint_forbidden_imports/forbidden_imports.py#L63
3. ...use it inside `visit*` method

Will you, please?

Reply:
Those exceptions are taken directly from google style guide ([test_reports_direct_imports](https://google.github.io/styleguide/pyguide.html#22-imports)). The task doesn't say anything about adding our own packages. We stick to google style.

Reply:
Oh right, that's what I was asking. Thanks for clarifying!